### PR TITLE
Added category Field 

### PR DESCRIPTION
**Pull request by Fredrick-Waihenya**
# Description
I've added a category  to the stori model that is a foreign key for the category model

### CODE REVIEW

**Review by JosephThuku — APPROVED** [review]
Will the userbe typing or selecting the category of the story

**Comment by Fredrick-Waihenya** [comment]
@JosephThuku  the user will be selecting from a list

File: blog/models.py [hunk 1]
```diff
@@ -22,7 +30,7 @@ class Stori(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
     status = models.IntegerField(choices=STATUS, default=0) #"""This here serves to indicate whether a stori has been published or not."""
-    #category 
+    category = models.ForeignKey(Category,on_delete=models.PROTECT)
```
Comment (JimmyTron) [inline blog/models.py:33]: why not have this display from the category model but indicates all the available options as a list???

**Review by JimmyTron — CHANGES_REQUESTED** [review]
check on this request

File: blog/models.py (hunk 1 above)
Comment (Fredrick-Waihenya) [reply blog/models.py:33]: i don't understand kindly expalin.

Comment (Fredrick-Waihenya) [reply blog/models.py:33]: If you add a category to the category model it will display as a list in the blog model.

Comment (JimmyTron) [reply blog/models.py:33]: yeah exactly

**Review by JimmyTron — APPROVED** [review]

**Pull request merged**